sig-windows/annual-report-2021.md

```diff
@@ -3,120 +3,82 @@
-
-   -
+   - yes
 
 5. Does the group have contributors from multiple companies/affiliations?
-
-   -
+   - yes
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
-
-   -
-   -
+   - testing hostProcess implementations on several windows apps
+   - improving our dev tools environment to grow the community
+   - hardening the CSI proxy and CSI support ecosystem
+   - performance testing Kubernetes on Windows extensively and publishing results
 
 ## Membership
-
-- Primary slack channel member count:
-- Primary mailing list member count:
-- Primary meeting attendee count (estimated, if needed):
-- Primary meeting participant count (estimated, if needed):
-- Unique reviewers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
-- Unique approvers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
+- Primary slack channel member count: 1507
+- Primary mailing list member count: 188
+- Primary meeting attendee count (estimated, if needed): 10
+- Primary meeting participant count (estimated, if needed): 10
+- Unique reviewers for SIG-owned packages: 6
+- Unique approvers for SIG-owned packages: 4
 
 Include any other ways you measure group membership
 
 ## Subprojects
 
-<!--
-In future, this will be generated from delta of sigs.yaml from $YYYY-01-01 to $YYYY-12-31
-Manually visible via `git diff HEAD@{$YYYY-01-01} HEAD@{$YYYY-12-31} -- $sig-id/README.md`
--->
-
-New in $YYYY:
-- [$subproject-name](https://git.k8s.io/community/$sig-id#$subproject-name)
--
-
-Retired in $YYYY:
-- [$subproject-name](https://git.k8s.io/community/$sig-id#$subproject-name)
--
+- windows csi-proxy subproject is active and healthy https://github.com/kubernetes-csi/csi-proxy
+    - meetings going well
+    - new optimization issue came up recently community engaged on it across companies vmware,rancher
 
-Continuing:
-- [$subproject-name](https://git.k8s.io/community/$sig-id#$subproject-name)
--
 
 ## Working groups
 
-<!--
-In future, this will be generated from delta of sigs.yaml from $YYYY-01-01 to $YYYY-12-31
-Manually visible via `git diff HEAD@{$YYYY-01-01} HEAD@{$YYYY-12-31} -- $sig-id/README.md`
--->
-
-New in $YYYY:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
-
-Retired in $YYYY:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
-
-Continuing:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
+Same as above.
 
 ## Operational
```

Comment:
can iterate on these in a follow-up in the next week or so, but the goal is to resolve them as part of closing https://github.com/kubernetes/community/issues/6386 to make sure these are accurate at least once a year